diff -Naru rt-thread_old/bsp/wch/arm/ch579m/board/drv_uart.c rt-thread/bsp/wch/arm/ch579m/board/drv_uart.c
--- rt-thread_old/bsp/wch/arm/ch579m/board/drv_uart.c	2022-09-22 15:59:01.155270700 +0800
+++ rt-thread/bsp/wch/arm/ch579m/board/drv_uart.c	2022-09-22 16:12:22.825428400 +0800
@@ -89,11 +89,11 @@
     switch (cfg->parity)
     {
     case PARITY_ODD:
-        R8_UART1_LCR |= R8_LCR_PAR_EN;
+        R8_UART1_LCR |= RB_LCR_PAR_EN;
         //R8_UART1_LCR |= 0x00;
         break;
     case PARITY_EVEN:
-        R8_UART1_LCR |= R8_LCR_PAR_EN;
+        R8_UART1_LCR |= RB_LCR_PAR_EN;
         R8_UART1_LCR |= 0x10;
         break;
     case PARITY_NONE:
